src/AssetBundle.php

```diff
@@ -78,7 +78,7 @@ class AssetBundle
      * ]
      * ```
      *
-     * @psalm-var array<array-key, string|CssFile>
+     * @var array<array-key, CssFile|string>
```

Comment:
Not good. `@var` is read by PhpStorm and, as far as I know, it can't read full-featured Psalm syntax.

Reply:
fixed